web/packages/plugins/apps/analysis-platform/src/utils/CRUD.tsx

```diff
@@ -0,0 +1,130 @@
+import { ANALYSIS_PATH, ASSET_PATH, DEFAULT_DATASOURCE_ID } from '../const'
```

Comment:
This entire file is unnecessary. :)

None of these wrapper functions provide much. Only less readable code. And more code to maintain.

Reply:
Not original author™, just ported old code (and extended based on previous work to ensure consistent pattern).